addons/website_forum/views/website_forum.xml

```diff
@@ -1415,7 +1415,7 @@
     <t t-if="allow_biography and object.can_display_biography" t-set="bio_popover_data">
         <div class="d-flex o_wforum_bio_popover_wrap">
             <img class="o_forum_avatar_big flex-shrink-0 mr-3" t-att-src="website.image_url(object.create_uid, 'image_128', '75x75')" alt="Avatar"/>
-            <div>
+            <div class="overflow-auto">
```

Comment:
If the fix is important, this should be done in scss as current user will not receive this fix otherwise. Not sure if this is a nice to have or a real problem?